lib/ibanity/error.rb

```diff
@@ -2,23 +2,26 @@ module Ibanity
   class Error < StandardError
     attr_reader :errors
 
-    def initialize(errors)
+    def initialize(errors, ibanity_request_id)
       @errors = errors
+      @ibanity_request_id = ibanity_request_id
     end
 
     def to_s
       if @errors.is_a?(Array) && @errors.size > 0
-        @errors.map do |error|
+        formatted_errors = @errors.map do |error|
           if error["meta"] && error["meta"]["attribute"]
-            "* #{error["code"]}: '#{error["meta"]["attribute"]}' #{error["detail"]}"
+           "* #{error["code"]}: '#{error["meta"]["attribute"]}' #{error["detail"]}"
           elsif error["meta"] && error["meta"]["resource"]
-            "* #{error["code"]}: '#{error["meta"]["resource"]}' #{error["detail"]}"
+           "* #{error["code"]}: '#{error["meta"]["resource"]}' #{error["detail"]}"
           else
-            "* #{error["code"]}: #{error["detail"]}"
+           "* #{error["code"]}: #{error["detail"]}"
           end
-        end.join("\n")
+        end
+            formatted_errors << "* ibanity_request_id: #{@ibanity_request_id}"
+        formatted_errors.join("\n")
       elsif @errors.is_a?(Hash)
-        "* #{@errors["error"]}: #{@errors["error_description"]}"
+        "* #{@errors["error"]}: #{@errors["error_description"]} * Error hint: #{errors["error_hint"]} * ibanity_request_id: #{@ibanity_request_id}"
```

Comment:
Could you format this with a `"\n"`  between the bullet points to be consistent with the array error format?

Reply:
Shouldn't this be `@errors["error_hint"]` instead of `errors["error_hint"]`?